README.md

```diff
@@ -1 +1,68 @@
-# conda-recipes
+# Conda Recipes
+[![Travis Build Status](https://travis-ci.org/RNAer/conda-recipes.png?branch=master)](https://travis-ci.org/RNAer/conda-recipes)
+
+Conda is a package manager application that quickly installs, runs, and updates packages and their dependencies. It was originally for Python virtual environment management, but it can essentially handle any type of packages in a isolated virtual environment.
```

Comment:
The second sentence is not exactly right as it was designed from the outset to be Python agnostic and manage complex dependencies. Keeping a sentence that mentions that it can keep packages and their dependencies in isolated virtual environments would be good.